specs/account_spec.rb

```diff
@@ -137,35 +159,72 @@
 end
 
 # TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Wave 2" do
+describe "Wave 2" do
+
+  before do
+    @id =[]
+    @balance =[]
+    @open_date =[]
+
+    CSV.open("../support/accounts.csv").each do |line|
+      @id << line[0].to_i
+      @balance << line[1].to_i
+      @open_date << line[2]
+    end
+
+  end
+
   describe "Account.all" do
+
     it "Returns an array of all accounts" do
       # TODO: Your test code here!
       # Useful checks might include:
       #   - Account.all returns an array
+      Bank::Account.all.class.must_equal Array
+
       #   - Everything in the array is an Account
       #   - The number of accounts is correct
-      #   - The ID and balance of the first and last
-      #       accounts match what's in the CSV file
+      number_of_accounts = 0
+      CSV.read("/Users/adai/Documents/ada/projects/BankAccounts/support/accounts.csv").each do |line|
+        number_of_accounts += 1
```

Comment:
You should change the path to: `support/accounts.csv` so that it can be run with rake from any person's computer and the path won't depend on your specific computer setup.